### PR TITLE
allows you to benchmark against 1 chosen URL.

### PR DESCRIPTION
**Pull request by lemire**
This modifies slightly one of our benchmark program so that we can narrowly benchmark URLs one by one if needed.

### CODE REVIEW

**Comment by anonrig** [comment]
aarch64 is once again failing with a constexpr issue...

File: benchmarks/wpt_bench.cpp [hunk 1]
```diff
@@ -7,7 +7,11 @@ double url_examples_bytes{};
 
 std::vector<std::pair<std::string, std::string>> url_examples;
 
-size_t init_data(const char *source) {
+enum {
+  ALL_URLS = -1,
```
Comment (anonrig) [inline benchmarks/wpt_bench.cpp:11]: Can you also add a documentation on how to run the benchmarks?

Comment (lemire) [reply benchmarks/wpt_bench.cpp:11]: @anonrig I pushed another commit which I expect might fix this build issue.

**Comment by lemire** [comment]
@anonrig Regarding documentation, have you seen this?

Screenshot 2024-09-28 at 3 53 17 PM


Note that though it is not well documented, we can also filter benchmarks.. 

```
./build/benchmarks/wpt_bench   tests/wpt/urltestdata.json --benchmark_filter=BasicBench_AdaURL_url_aggregator
```

But that's Google Benchmark stuff. (And it was not previously documented.)

Note that the command line processing is not great in these tools. It is kind of a hack, but we don't expect most users to care about these details.

**Comment by lemire** [comment]
I see it is still failing the aarch64 ... :-/

**Review by anonrig — APPROVED** [review]

**Comment by anonrig** [comment]
@lemire it seems aarch64 is still failing. would you mind taking a look into it?

**Pull request merged**